### PR TITLE
fix: revert modifications to Apache license

### PR DESCRIPTION
**Pull request by svrnm**
# Description

See open-telemetry/community#2056 for background

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

not applicable

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

- [N/A] Followed the style guidelines of this project
- [ ] Changelogs have been updated -- please let me know if a changelog update is required and I will provide it in the PR
- [N/A] Unit tests have been added
- [N/A] Documentation has been updated

### CODE REVIEW

**Review by lzchen — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright The OpenTelemetry Authors
```
Comment (ocelotl) [inline LICENSE:189]: This line exists in many other files too.

Comment (ocelotl) [reply LICENSE:189]: Do we need to change them as well?

Comment (svrnm) [reply LICENSE:189]: yes, but I can update the PR to fix them as well since I have done the same for other projects already, let me look into that.

Comment (svrnm) [reply LICENSE:189]: see https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2429/commits/30e8476b4b1eb560249c7960a13dc13b899d9e96

File: LICENSE.Apache [hunk 2]
```diff
@@ -178,15 +178,16 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
```
Comment (arminru) [inline LICENSE.Apache:181]: Huh, any idea why this change was made? Could there be a purpose we're not aware of for this `LICENSE.Apache` file?

**Review by arminru — APPROVED** [review]

File: LICENSE.Apache (hunk 2 above)
Comment (svrnm) [reply LICENSE.Apache:181]: There are 2 versions of the Apache license one use `{}` the other one uses `[]`, I replaced all Apache licenses with the same file with `[]` that's why this change was made

**Review by ocelotl — APPROVED** [review]
